public/v4/apps/photoview.yml

```diff
@@ -0,0 +1,93 @@
+captainVersion: 4
+
+services:
+    $$cap_appname-mariadb:
+        image: mariadb:$$cap_mariadb_version
+        environment:
+            MYSQL_RANDOM_ROOT_PASSWORD: '1'
+            MYSQL_DATABASE: $$cap_mariadb_db
+            MYSQL_USER: $$cap_mariadb_user
+            MYSQL_PASSWORD: $$cap_mariadb_pass
+        volumes:
+            - $$cap_appname-mariadb-data:/var/lib/mysql
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+
+    $$cap_appname:
+        image: viktorstrate/photoview:$$cap_photoview_version
+        depends_on:
+            - $$cap_appname-mariadb
+        environment:
+            PHOTOVIEW_DATABASE_DRIVER: mysql
+            PHOTOVIEW_MYSQL_URL: $$cap_mariadb_user:$$cap_mariadb_pass@tcp(srv-captain--$$cap_appname-mariadb)/$$cap_mariadb_db
+            PHOTOVIEW_LISTEN_IP: 0.0.0.0
+            PHOTOVIEW_LISTEN_PORT: 80
+            PHOTOVIEW_MEDIA_CACHE: /app/cache
+            MAPBOX_TOKEN: $$cap_mapbox_token
+        volumes:
+            - $$cap_appname-media-cache:/app/cache
+            - $$cap_photos_path:/photos:ro
```

Comment:
`:ro` suffix isn't parsed by CapRover. Must be removed.

Reply:
Done.